From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jastru__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

Commit:
device-management-python intial draft

Commit:
fixed links, swapped section

carolinacmoravia committed:
M21606: Potential duplicated content: "scheduled messages"

Hello, @christianwolf42,

Translator has reported possible source content issue. It seems that "scheduled messages" is duplicated.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

carolinacmoravia committed:
C11432: Extra space in bold type

Hello, @bradsev,

Localization team has reported source content issue that causes issue for localization. 

Please, help to check my comment tag added into the article and reply with explanation if source text should be fixed, or it is by design. If fix is needed, please, let me know either if you would like me to fix it in this PR, or if you will fix it in another PR. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

chetansriv committed:
Tutorial custom apps

chetansriv committed:
Tutorial custom apps update

genlin committed:
add a article

genlin committed:
update the article

genlin committed:
format the article

genlin committed:
update TOC

genlin committed:
update the date

carolinacmoravia committed:
M22773: Misleading sentence 

Hello, @jeevansd, 

Translator has reported possible source content issue. It seems that sentence "to link your application to ADP Federated SSO." should be concatenated with the previous sentence.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

carolinacmoravia committed:
M22774: Unnecessary word:  "either"

Hello, @thraka, 

Translator has reported possible source content issue.  It seems that "either" is unnecssary. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

barclayn committed:
deleting the azure security and az log article and removing entries in toc files pointing to it

GitHubber17 committed:
edit pass: install-vm-agent-offline

tankungx committed:
update data lake vscode doc

carolinacmoravia committed:
C11873: Extra asterisk breaking format 

Hello, @jimdial, 

Localization team has reported source content issue that causes issue for localization. 
 
Please, help to check my comment tag added into the article and reply with explanation if source text should be fixed, or it is by design. If fix is needed, please, let me know either if you would like me to fix it in this PR, or if you will fix it in another PR. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

GitHubber17 committed:
edit pass: install-vm-agent-offline

karthikananth committed:
Update expressroute-locations-providers.md

jodebrui committed:
adding link to HA article

Commit:
Updating several email addresses to be in correct format.

PRMerger20 committed:
Merge pull request #33074 from karthikananth/patch-24

Update expressroute-locations-providers.md

Commit:
quickfix06

v-geberr committed:
remove luis.ai - use regions reference page

v-geberr committed:
edits

Commit:
Merge pull request #33075 from jodebrui/patch-11

adding link to HA article

Commit:
quickfix06

PRMerger12 committed:
Merge pull request #33079 from v-geberr/0208-region-fix-1

0208 region fix 1

Commit:
quickfix06

spelluru committed:
updated images

PRMerger19 committed:
Merge pull request #33077 from johndeu/master

Updating several email addresses to be in correct format.

Ja-Dunn committed:
Merge pull request #33080 from spelluru/BulkCopy0208

updated images

PRMerger20 committed:
Merge pull request #33078 from MarkusVi/quickfix06

quickfix06

fauhse committed:
Updated Version table to include R1 agent version.

sanjeev3 committed:
Update Text-Moderation-API-Languages.md

Commit:
quickfix41

sanjeev3 committed:
Update Text-Moderation-API-Languages.md

sanjeev3 committed:
Update try-text-api.md

sanjeev3 committed:
Update text-moderation-api.md

barclayn committed:
adding redirect entry

PRMerger18 committed:
Merge pull request #33083 from sanjeev3/master

Text API doc updates (minor)

Commit:
Merge pull request #33084 from fauhse/afs-R1

Updated Version table to include R1 agent version.

Commit:
quickfix51

Commit:
quickfix51

PRMerger18 committed:
Merge pull request #33082 from MarkusVi/quickfix41

quickfix41

GitHubber17 committed:
Merge pull request #33013 from barclayn/azlog-asc-delete

deleting the azure security and az log article and removing entries i…

genlin committed:
fix the block issue

GitHubber17 committed:
Merge pull request #33085 from MarkusVi/quickfix51

quickfix51

genlin committed:
fix the typo in smtp

GitHubber17 committed:
Merge pull request #32896 from genlin/master557

create-ptr-for-smtp-service.md

tankungx committed:
update data lake for vscode doc

tankungx committed:
update data lake vscode doc

genlin committed:
add a new article

genlin committed:
update toc

genlin committed:
correct typo

genlin committed:
upload images

genlin committed:
correct typo

genlin committed:
add image

genlin committed:
update images

genlin committed:
update the the date

giladmit committed:
small changes

carolinacmoravia committed:
Update service-bus-geo-dr.md

PRMerger13 committed:
Merge pull request #33097 from giladmit/GiladWorkingBranch

small changes

PRMerger20 committed:
Merge pull request #32649 from carolinacmoravia/patch-593

C11432: Extra space in bold type

PRMerger10 committed:
Merge pull request #32905 from carolinacmoravia/patch-602

M22773: Misleading sentence

PRMerger12 committed:
Merge pull request #32906 from carolinacmoravia/patch-603

M22774: Unnecessary word:  "either"

Commit:
Merge pull request #33059 from carolinacmoravia/patch-611

C11873: Extra asterisk breaking format

PRMerger18 committed:
Merge pull request #32148 from carolinacmoravia/patch-561

M21606: Potential duplicated content: "scheduled messages"

GitHubber17 committed:
Merge pull request #33036 from tankungx/USQLVSCODENEW

update data lake vscode doc

v-geberr committed:
corrections

itechedit committed:
Merge pull request #33094 from genlin/master557

vpn-gateway-radiuis-mfa-nsp.md

WilliamAntonRohm committed:
resized images - batch 01 of 09

Commit:
add pip install packages

WilliamAntonRohm committed:
resized images - batch 02 of 09

WilliamAntonRohm committed:
resized images - batch 03 of 09

WilliamAntonRohm committed:
resized images - batch 04 of 09

WilliamAntonRohm committed:
resized images - batch 05 of 09

v-geberr committed:
region fix

ShawnJackson committed:
Update install-vm-agent-offline.md

WilliamAntonRohm committed:
resized images - batch 06 of 09

WilliamAntonRohm committed:
resized images - batch 07 of 09

WilliamAntonRohm committed:
resized images - batch 08 of 09

WilliamAntonRohm committed:
resized images - batch 09 of 09

Commit:
fix format, link

itechedit committed:
Merge pull request #33104 from WilliamAntonRohm/W-hdi-images-01

resized images - batch 01 of 09

itechedit committed:
Merge pull request #33105 from WilliamAntonRohm/W-hdi-images-02

resized images - batch 02 of 09

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

itechedit committed:
Merge pull request #33106 from WilliamAntonRohm/W-hdi-images-03

resized images - batch 03 of 09

itechedit committed:
Merge pull request #33107 from WilliamAntonRohm/W-hdi-images-04

resized images - batch 04 of 09

itechedit committed:
Merge pull request #33108 from WilliamAntonRohm/W-hdi-images-05

resized images - batch 05 of 09

itechedit committed:
Merge pull request #33109 from WilliamAntonRohm/W-hdi-images-06

resized images - batch 06 of 09

itechedit committed:
Merge pull request #33111 from WilliamAntonRohm/W-hdi-images-07

resized images - batch 07 of 09

itechedit committed:
Merge pull request #33112 from WilliamAntonRohm/W-hdi-images-08

resized images - batch 08 of 09

itechedit committed:
Merge pull request #33113 from WilliamAntonRohm/W-hdi-images-09

resized images - batch 09 of 09

ShawnJackson committed:
Merge pull request #33028 from GitHubber17/vsts-1180012

edit pass: install-vm-agent-offline

PRMerger10 committed:
Merge pull request #33110 from v-geberr/0209-luis.ai-to-region

region fix

PRMerger14 committed:
Merge pull request #33102 from v-geberr/0209-boundaries-fix

corrections

rmca14 committed:
Merge pull request #32661 from chetansriv/customapp

Tutorial custom apps

itechedit committed:
Merge pull request #31407 from msebolt/device-management-python

device-management-python intial draft

v-alje committed:
Merge pull request #33117 from MicrosoftDocs/FromPublicRepo

From public repo
